requirements/test.txt

```diff
@@ -6,3 +6,4 @@ pytest>=4.6 # required for pytest-cov
 pytest-django
 pytest-cov
 wheel
+mysqlclient # tested against in CI for now
```

Comment:
Will we need this for getting 2.8 mysql production synced to a 2.8.1 postgres production, or are the expected changes for 2.8.1 minimal enough we won't need to deploy it to the old production server before migrating?

(otherwise, seems like a great solution)

Reply:
I think the only other change for 2.8.1 is a bug that's a one-line change (have solution locally), so seems like the latter.